acl/types.go

```diff
@@ -25,3 +25,29 @@ const (
 	// EACLPublicAppendRule is a basic ACL value for non-final public-append container for which extended ACL CAN be set.
 	EACLPublicAppendRule = 0x0FBF9FFF
 )
+
+const (
+	// PublicBasicName is a well-known name for 0x1FBFBFFF basic ACL.
+	PublicBasicName = "public-read-write"
+
+	// PrivateBasicName is a well-known name for 0x1C8C8CCC basic ACL.
```

Comment:
How about shortly describing the meaning of ACL in simple terms?

Reply:
Added comment for `BasicACL` type.

Reply:
I've meant descriptions like `PrivateBasicName represents fully-private container without eACL`.